mount_test.go

```diff
@@ -111,26 +111,26 @@ func Test_App_Mount_Express_Behavior(t *testing.T) {
 
 	app := New()
 	subApp := New()
+
 	// app setup
-	{
```

Comment:
this kind of encapsulation i have copied from your pull requests^^ and now you remove it

Why not keep this grouping


https://github.com/gofiber/fiber/pull/2288/files#diff-654acee0bb6ef56dde4305631ed3d98769cfea139f70647ad580d5fe4737c4ec

https://github.com/gofiber/fiber/pull/2314/files#diff-5c85ef45029b22bed483baf176606b6619b80a9a470dbe7f64a3a9a98cba8161

Reply:
Creating a new scope only makes sense if new variables are created within it, which should not be available in the outer scope. As no such new var is created here, we can simply drop the new scope (which even costs a bit of CPU cycles if not optimized away by the compiler).